src/main/kotlin/no/nav/familie/ef/sak/behandling/BehandlingService.kt

```diff
@@ -209,9 +209,22 @@ class BehandlingService(
             utfall = StegUtfall.HENLAGT,
             metadata = henlagt
         )
+        opprettStatistikkTask(henlagtBehandling)
         return behandlingRepository.update(henlagtBehandling)
     }
 
+    private fun opprettStatistikkTask(behandling: Behandling) {
+        if (!behandling.erMigrering()) {
+            taskService.save(
+                BehandlingsstatistikkTask.opprettHenlagtTask(
+                    behandlingId = behandling.id,
+                    hendelseTidspunkt = behandling.sporbar.endret.endretTid,
```

Comment:
endretTid her kan være når behandlingen ble opprettet for noen dager sen, så tror vi kan bruke `now()`?

Reply:
Enig - og da trenger vi ikke å bekymre oss for duplicate heller